avaframe/com1DFA/com1DFA.py

```diff
@@ -1604,14 +1608,21 @@ def initializeResistance(cfg, dem, simTypeActual, resLine, reportAreaInfo, thres
     -------
     cResRaster : 2D numpy array
         raster of resistance coefficients
+    detRaster : 2D numpy array
+        raster of detrainment coefficients
     reportAreaInfo: dict
         simulation area information dictionary completed with entrainment area info
     """
     cRes = cfg.getfloat("cRes")
+    K = cfg.getfloat("detK")
+    DetBool = cfg.getboolean("DetBool")
```

Comment:
Please rename this to just Detrainment or similar.  There's no need to name the variable after it's type (see here: https://github.com/avaframe/AvaFrame/issues/330 ; i.e. we try to avoid this)